examples/StreetlightsAPI.TopLevelStatement/StreetlightsAPI.TopLevelStatement.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <!-- This project is targeting .NET8 intentionally (The 'old school' Startup-class project is .NET6), to prove that
+         the AsyncAPI.Saunter.Generator.Cli tool can generate specs for projects targetting .NET6 and .NET8. -->
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Added 2nd example project (also to be used in integration test) because Top Level Statement projects initially didn't work with the Generator.Cli tool. Now it does work too. And proves using integration test that both NET6 & NET8 targets work.